client/src/addSighting.jsx

```diff
@@ -32,12 +32,17 @@ export class NewSighting extends React.Component {
 		event.preventDefault();
 
 		//Move input from user into single JSON
+		const user = localStorage.getItem('user');
+		const userID = user._id;
 		const newSighting = {
 			animal: this.state.animal,
 			location: this.state.location,
-			time: new Date()
+			time: new Date(),
+			spottedBy: userID
```

Comment:
This is submitting with `spottedBy` as undefined when I'm logged in.
JSON in localstorage is stored as a string, so you have to `JSON.parse` the result

Reply:
Updated to remove `spottedBy` from addSighting.jsx. Now pulls from `req.user` using passport's sessions.